doc/src/cylc-user-guide/cug.tex

```diff
@@ -381,7 +381,9 @@ \section{Cylc Screenshots}
 \section{Installation}
 \label{Requirements}
 
-Cylc runs on Unix variants, usually Linux, and including Apple OS X.
+Cylc runs on Linux. It is tested quite thoroughly on modern RHEL and Ubuntu
+distros. Some users have also managed to make it work on other Unix variants
+including Apple OS X.
```

Comment:
> Some users have also managed to make it work on other Unix variants including Apple OS X.

That phrasing sounds to me like *It is working under OSX because some users could make it to work*. However, it is unfortunately not documented nor obvious how one can make it work under OSX (see https://github.com/cylc/cylc/issues/2689), so I would rather phrase it more transparent, e.g.

> Some users have also managed to make it work on other Unix variants including Apple OS X, but they are not officially tested and supported.

Reply:
I have added the suggested change to the latest commit.